### ISSUE TITLE
Add support for Ubuntu 22.04 Jammy Jellyfish

### ISSUE THREAD

**Issue by lk86** [MinaProtocol/mina#10752]

**Comment by lk86** [comment]
no-install-recommends lays the groundwork for 22.04 support but the official release is not ready yet and packages from the "jammy" repo do not seem to work properly yet. Once this is fixed upstream, create `buildkite/src/Jobs/Release/MinaArtifactJammy.dhall` in the style of the other artifacts

**Comment by NessDan** [comment]
Please, I just did a fresh install of Ubuntu 22.04 Jammy and now I can't get Mina running.

**Comment by Nasfame** [comment]
+1 for jammy relates to #14546

**Comment by leprick0n** [comment]
+1 for requesting the Mina mainnet binary for 22.04 and 24.04

**Comment by glyh** [comment]
Not for Jammy, but there's on-going effort to support 24 Noble
https://github.com/MinaProtocol/mina/pull/17267
